website/client/router.js

```diff
@@ -101,6 +103,14 @@ export default new VueRouter({
             },
           ],
         },
+        { name: 'challenges', path: 'challenges', component: Page },
+      ],
+    },
+    {
+      path: '/user',
```

Comment:
Yea, I think I just messed up when merging the routes. I'll take a look at develop and fix it.